mne/viz/_3d.py

```diff
@@ -1158,6 +1249,285 @@ def time_label(t):
 
     if initial_time is not None:
         brain.set_time(initial_time)
+
+    _toggle_mlab_render(brain._figures, True)
+
+    if time_viewer:
+        TimeViewer(brain)
+
+    return brain
+
+
+def plot_vector_source_estimates(stc, subject=None, hemi='lh', colormap='auto',
+                                 time_label='auto', smoothing_steps=10,
+                                 transparent=None, alpha=1.0, vector_alpha=1.0,
```

Comment:
the `alpha=1.0` parameter is the alpha at which the texture is drawn onto the brain. Then, the brain alpha is tweaked with `brain_alpha=0.4`, which includes its texture. This results IMO in pretty good visibility / aesthetics. I'm open for suggestions for other values :) However, things may change completely if we go the PR to pysurfer route.